### PR TITLE
SAML Test

### PR DESCRIPTION
**Pull request by jakobvogel**

### CODE REVIEW

**Review by hggh — APPROVED** [review]

File: src/test/java/sirius/web/security/SamlSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.web.security
+
+import sirius.kernel.BaseSpecification
+import sirius.kernel.di.std.Part
+import sirius.kernel.health.HandledException
+
+class SamlSpec extends BaseSpecification {
```
Comment (andyHa) [inline src/test/java/sirius/web/security/SamlSpec.groovy:15]: could this be a kotlin test (and use multiline string literals?)

File: src/test/kotlin/sirius/web/security/SamlTest.kt [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+                response.fingerprint
+        )
+        assertEquals(
+                "https://sso.hq.scireum.com/adfs/services/trust",
+                response.issuer
+        )
+    }
+
+    @Test
+    fun `cidaas SAML response fails correctly when ignoring time`() {
```
Comment (sabieber) [inline src/test/kotlin/sirius/web/security/SamlTest.kt:98]: Think this test was named incorrectly in the groovy version

File: src/test/kotlin/sirius/web/security/SamlTest.kt [hunk 3]
```diff
@@ -56,31 +56,32 @@ class SamlTest {
...
...ResponseTo="identifier_1652353644351"><Issuer xmlns="urn:oasis:names:tc:SAML:2.0:assertion">https://sso.hq.scireum.com/adfs/services/trust</Issuer><samlp:Status><samlp:StatusCode Value="urn:oasis:names:tc:SAML:2.0:status:Success"/></samlp:Status><Assertion xmlns="urn:oasis:names:tc:SAML:2.0:assertion" ID="_30e5c8bb-ee3a-4290-9960-f5fae240d0e0" IssueInstant="2022-05-12T11:07:26.759Z" Version="2.0"><Issuer>https://sso.hq.scireum.com/adfs/services/trust</Issuer><ds:Signature xmlns:ds="http://www.w3.org/2000/09/xmldsig#"><ds:SignedInfo><ds:CanonicalizationMethod Algorithm="http://www.w3.org/2001/10/xml-exc-c14n#"/><ds:SignatureMethod Algorithm="http://www.w3.org/2001/04/xmldsig-more#rsa-sha256"/><ds:Reference URI="#_30e5c8bb-ee3a-4290-9960-f5fae240d0e0"><ds:Transforms><ds:Transform Algorithm="http://www.w3.org/2000/09/xmldsig#enveloped-signature"/><ds:Transform Algorithm="http://www.w3.org/2001/10/xml-exc-c14n#"/></ds:Transforms><ds:DigestMethod Algorithm="http://www.w3.org/2001/04/xmlenc#sha256"/><ds:DigestValue>oSkdX5u1Xt5PoFYL4HB+3iN9Y5Flmo8PJcL72gLBPr0=</ds:DigestValue></ds:Reference></ds:SignedInfo><ds:SignatureValue>wK618x44fMUudGg4xnGw8gETbVxMYGv73YMpGuD5gS8mOcU1xS9t2gninPexZEzh
```
Comment (sabieber) [inline src/test/kotlin/sirius/web/security/SamlTest.kt:60]: Tried to improve the display/indentation of these XML blocks but when formatting them with IntelliJ or VS Code and putting the result into the string literals the tests fail.

**Review by sabieber — APPROVED** [review]

**Comment by jakobvogel** [comment]
Thanks very much @sabieber!

**Pull request merged**